moveit_msgs/CollisionObject object
bool add  # true = add, false = remove
---
bool success
string message
